src/components/views/settings/notifications/NotificationSettings2.tsx

```diff
@@ -0,0 +1,368 @@
+/*
+Copyright 2023 The Matrix.org Foundation C.I.C.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React, { useState } from "react";
+
+import NewAndImprovedIcon from "../../../../../res/img/element-icons/new-and-improved.svg";
+import { useMatrixClientContext } from "../../../../contexts/MatrixClientContext";
+import { useNotificationSettings } from "../../../../hooks/useNotificationSettings";
+import { useSettingValue } from "../../../../hooks/useSettings";
+import { _t } from "../../../../languageHandler";
+import {
+    DefaultNotificationSettings,
+    NotificationSettings,
+} from "../../../../models/notificationsettings/NotificationSettings";
+import { RoomNotifState } from "../../../../RoomNotifs";
+import { SettingLevel } from "../../../../settings/SettingLevel";
+import SettingsStore from "../../../../settings/SettingsStore";
+import { NotificationColor } from "../../../../stores/notifications/NotificationColor";
+import { clearAllNotifications } from "../../../../utils/notifications";
+import AccessibleButton from "../../elements/AccessibleButton";
+import LabelledCheckbox from "../../elements/LabelledCheckbox";
+import LabelledToggleSwitch from "../../elements/LabelledToggleSwitch";
+import StyledRadioGroup from "../../elements/StyledRadioGroup";
+import TagComposer from "../../elements/TagComposer";
+import { StatelessNotificationBadge } from "../../rooms/NotificationBadge/StatelessNotificationBadge";
+import { SettingsBanner } from "../shared/SettingsBanner";
+import { SettingsSection } from "../shared/SettingsSection";
+import SettingsSubsection from "../shared/SettingsSubsection";
+import { NotificationPusherSettings } from "./NotificationPusherSettings";
+
+enum NotificationDefaultLevels {
+    ALL_MESSAGES = "ALL_MESSAGES",
+    PEOPLE_MENTIONS_KEYWORDS = "PEOPLE_MENTIONS_KEYWORDS",
+    MENTIONS_KEYWORDS = "MENTIONS_KEYWORDS",
+}
```

Comment:
Can we follow the code style for enums please? https://github.com/vector-im/element-web/blob/develop/code_style.md#typescript--javascript 7-8